src/mlpack/bindings/go/generate_cpp.cpp.in

```diff
@@ -0,0 +1,49 @@
+/*
+* @file generate_cpp_${PROGRAM_NAME}.cpp
+* @author Yasmine Dumouchel
+*
+* This is an automatically-generated file that is used to generate the .cpp
+* files that are used for the Go bindings.  This program will print the
+* .cpp file on stdout when run and doesn't need any input parameters.
+*
+* The CMake variable ${PROGRAM_NAME} must be set for
+* this to configure correctly.
+*
+* mlpack is free software; you may redistribute it and/or modify it under the
+* terms of the 3-clause BSD license.  You should have received a copy of the
+* 3-clause BSD license along with mlpack.  If not, see
+* http://www.opensource.org/licenses/BSD-3-Clause for more information.
+*/
```

Comment:
Little comment, these lines should be spaced in one, e.g.,

```
/*
 * (comment)
 */
```

not

```
/*
* (comment)
*/
```

Reply:
Resolved.